src/cppcache/src/MapSegment.hpp

```diff
@@ -164,9 +166,9 @@ class CPPCACHE_EXPORT MapSegment {
     m_entryFactory->newMapEntry(key, newEntry);
     newEntry->setValueI(newValue);
     if (m_concurrencyChecksEnabled) {
-      if (versionTag != NULLPTR && versionTag.ptr() != NULL) {
+      if (versionTag != NULLPTR && versionTag.ptr() != nullptr) {
```

Comment:
`NULLPTR` is Geode constant from "empty" `SharedPtr`. 

`nullptr` is C++11 for "empty" pointer.